samples/MauiEmbedding/MauiEmbedding/MauiEmbedding.csproj

```diff
@@ -1,69 +1,51 @@
-<Project Sdk="Microsoft.NET.Sdk">
-	<!-- This is only included because the project references Uno.Extensions.Maui.UI as project reference -->
-	<Import Project="..\..\..\src\Uno.Extensions.Maui.UI\build\Package.props" />
+<Project Sdk="Uno.Sdk">
+	<Import Project="..\..\..\src\tfms-ui-maui.props" />
 
-	<PropertyGroup>
-		<TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows')) or '$(EnableWindowsTargeting)' == 'true'">$(TargetFrameworks);net8.0-windows10.0.19041</TargetFrameworks>
-		<TargetFrameworks>$(TargetFrameworks);net8.0;net8.0-ios;net8.0-android;net8.0-maccatalyst</TargetFrameworks>
+  <PropertyGroup>
+  
+    <OutputType>Exe</OutputType>
+    <UnoSingleProject>true</UnoSingleProject>
 
-		<!-- Ensures the .xr.xml files are generated in a proper layout folder -->
-		<GenerateLibraryLayout>true</GenerateLibraryLayout>
-		<IsMauiEmbedding Condition="'$(IsMauiEmbedding)'==''">false</IsMauiEmbedding>
-	</PropertyGroup>
+    <!-- Display name -->
+    <ApplicationTitle>MauiEmbedding</ApplicationTitle>
+    <!-- App Identifier -->
+    <ApplicationId>com.companyname.MauiEmbedding</ApplicationId>
+    <!-- Versions -->
+    <ApplicationDisplayVersion>1.0</ApplicationDisplayVersion>
+    <ApplicationVersion>1</ApplicationVersion>
+    <!--
+      If you encounter this error message:
 
-	<ItemGroup>
-		<PackageReference Include="Uno.Dsp.Tasks" />
-		<PackageReference Include="Uno.WinUI" />
-		<PackageReference Include="Uno.Resizetizer" />
-		<PackageReference Include="CommunityToolkit.Mvvm" />
-		<PackageReference Include="Uno.Material.WinUI" />
-		<PackageReference Include="Uno.Toolkit.WinUI.Material" />
-		<PackageReference Include="Uno.Toolkit.WinUI" />
-		<PackageReference Include="Microsoft.Extensions.Logging.Console" />
-	</ItemGroup>
+        error NETSDK1148: A referenced assembly was compiled using a newer version of Microsoft.Windows.SDK.NET.dll.
+        Please update to a newer .NET SDK in order to reference this assembly.
 
-	<Choose>
-		<When Condition="$(IsWinAppSdk)">
-			<PropertyGroup>
-				<!--
-				If you encounter this error message:
+      This means that the two packages below must be aligned with the "build" version number of
+      the "Microsoft.Windows.SDK.BuildTools" package above, and the "revision" version number
+      must be the highest found in https://www.nuget.org/packages/Microsoft.Windows.SDK.NET.Ref.
+    -->
+    <!-- <WindowsSdkPackageVersion>10.0.22621.28</WindowsSdkPackageVersion> -->
 
-					error NETSDK1148: A referenced assembly was compiled using a newer version of Microsoft.Windows.SDK.NET.dll.
-					Please update to a newer .NET SDK in order to reference this assembly.
+    <!--
+      UnoFeatures let's you quickly add and manage implicit package references based on the features you want to use.
+      https://aka.platform.uno/singleproject-features
+    -->
+    <UnoFeatures>
+      Material;
+      Dsp;
+      Toolkit;
+      Mvvm;
+      ThemeService;
```

Comment:
Wait doesn't this bring in the Extensions CoreUI Package?

Reply:
Removed ThemeService